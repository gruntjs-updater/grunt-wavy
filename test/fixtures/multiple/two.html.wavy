<div class="multiple">
    @include test(Two Test)
</div>