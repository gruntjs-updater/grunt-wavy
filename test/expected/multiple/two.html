<div class="multiple">
    
    <div>
    Two Test
    </div>

</div>